Dockerfile.dev

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get install -y curl
+RUN curl https://get.volta.sh | bash
```

Comment:
If we're using docker, it's better to directly use the official node image instead of running ubuntu, installing volta and then installing node. Maybe an `alpine` or `slim` build, whichever doesn't break any dependencies. 

https://hub.docker.com/_/node

Reply:
Fully agree on that but is was not clear how far should I step aside from current guidelines, so I decided to stick to them as close as possible. I'll start implementing some diagrams logic, and then it will be clear if it is convenient to do via Docker

Reply:
Fixed